src/components/presenter-mode/index.js

```diff
@@ -108,15 +113,17 @@ export default function PresenterMode(props) {
         >
           {children}
         </Deck>
-        <Deck
-          disableInteractivity
-          useAnimations={false}
-          backdropStyle={deckBackdropStyles.nextSlide}
-          ref={previewDeck}
-          theme={theme}
-        >
-          {children}
-        </Deck>
+        <PreviewSlideWrapper visible={showFinalSlide}>
```

Comment:
The reason we use css `visibility` over not rendering it is we need to keep the `ref` to the preview deck always active.